onnxruntime/contrib_ops/cuda/bert/group_query_attention_impl.cu

```diff
@@ -574,13 +574,13 @@ Status LaunchUnpackQKV(const T* packed_qkv, T* unpacked_q, T* unpacked_k, T* unp
 }
 
 // Kernel to convert seqlens_k to position_ids
-__global__ void SeqlensToPosIdsPrompt(int32_t* seqlens_k, int64_t* position_ids, const int seqlen,
+__global__ void SeqlensToPosIdsPrompt(int64_t* position_ids, const int seqlen,
                                       const int batch_size) {
   int tid = blockDim.x * blockIdx.x + threadIdx.x;
   int b = tid / seqlen;
   int s = tid % seqlen;
   if (b < batch_size) {
-    if (s < seqlens_k[b] + 1) {
+    if (s < seqlen) {
```

Comment:
Does it mean the padding token is also assigned a position id?
Is it compatible with huggingface implementation?

Reply:
very true, will revert for now